hypeapi/fixtures/users.json

```diff
@@ -0,0 +1,303 @@
+[
```

Comment:
It looks like this is specific to your database. I'm not sure what this is but it should probably be put in the .gitignore